Src/FluentAssertions/Types/TypeSelector.cs

```diff
@@ -183,6 +183,26 @@ public TypeSelector ThatAreNotClasses()
         return this;
     }
 
+    /// <summary>
+    /// Determines the type is abstract and must be overridden
+    /// </summary>
+    /// <returns></returns>
+    public TypeSelector ThatAreAbstract()
+    {
+        types = types.Where(t => t.IsAbstract).ToList();
```

Comment:
You'll need to use `t.IsCSharpAbstract()` instead of `t.IsAbstract`.

A `static` type in C# also returns true for `IsAbstract` in reflection.
[Example](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEUCuA7AHwAEBGAOlIE4AKDATwAcYIAzagZQwEMMBLMAMIAbLgGdRASjIBJUQEFgojFC5gMEgNwBYAFC7SSAAREATIc49+wsaIDeAXyA=)